app/src/main/res/xml/backup_rules.xml

```diff
@@ -1,19 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!--
-     Copyright 2023 The Duckee Project
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-          http://www.apache.org/licenses/LICENSE-2.0
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
 <full-backup-content>
-    <!--
-   <include domain="sharedpref" path="."/>
-   <exclude domain="sharedpref" path="device.xml"/>
--->
-</full-backup-content>
+    <exclude domain="sharedpref" path="airbridge-internal" />
+    <exclude domain="sharedpref" path="airbridge-install" />
+    <exclude domain="sharedpref" path="airbridge-user-info" />
+    <exclude domain="sharedpref" path="airbridge-user-alias" />
+    <exclude domain="sharedpref" path="airbridge-user-attributes" />
+    <exclude domain="database" path="airbridge.db" />
+</full-backup-content>
```

Comment:
Added Airbridge SDK-specific backup rules to prevent automatic backup of internal data. This is crucial for maintaining data integrity and privacy. The Airbridge SDK documentation states that these rules are necessary to exclude sensitive data from Android's Auto Backup feature. Without these rules, there might be conflicts with the SDK's internal data management.